contracts/interfaces/random-beacon/IRandomBeacon.sol

```diff
@@ -0,0 +1,194 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.9;
+
+import { RandomRequest } from "../../libraries/LibSLA.sol";
+import { VRF } from "@chainlink/contracts/src/v0.8/VRF.sol";
+
+interface IRandomBeacon {
+  /// @dev Throws if current period is less than the target activation period
+  error ErrNotActivated(uint256 untilPeriod);
+  /// @dev Throws if the cool down for key hash change is not ended
+  error ErrKeyHashChangeCooldownNotEnded();
+  /// @dev Throws if the cool down for registration is not ended
+  error ErrRegisterCoolDownNotEnded();
+  /// @dev Throws if the proof is invalid
+  error ErrInvalidProof();
+  /// @dev Throws if the request is not finalized
+  error ErrBeaconNotFinalized(uint256 period);
+  /// @dev Throws if the period is invalid (too early or too late)
+  error ErrInvalidPeriod();
+  /// @dev Throws if the request is already submitted
+  error ErrAlreadySubmitted();
+  /// @dev Throws if the request is already finalized
+  error ErrBeaconAlreadyFinalized(uint256 period);
+  /// @dev Throws if the random request is unexists
+  error ErrInvalidRandomRequest(bytes32 expected, bytes32 got);
+  /// @dev Throws if the key hash is not match with the one in the profile
+  error ErrInvalidKeyHash(bytes32 expected, bytes32 actual);
+
+  /**
+   * @dev The validator type.
+   */
+  enum ValidatorType {
+    Unknown,
+    // Max Validator Number
+    All,
```

Comment:
Is it All = Governing + Standard + Rotating?

Reply:
All is not equal to `Governing` + `Standard` + `Rotating`

Reply:
Yes, expected it to be